### PR TITLE
Typo "Microsoft Teams Room"→"Microsoft Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/microsoftteams/rooms/content-camera
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/content-camera.md
#PingMSFTDocs

### CODE REVIEW

File: Teams/rooms/content-camera.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ ms.collection:
   - Teams_ITAdmin_Rooms
 ms.custom: 
   - seo-marvel-mar2020
-description: Use a content camera in a Microsoft Teams Room, which interacts with image-processing software to allow presenters to draw on an analog whiteboard.
+description: Use a content camera in a Microsoft Teams Rooms, which interacts with image-processing software to allow presenters to draw on an analog whiteboard.
```
Comment (JohanFreelancer9) [inline Teams/rooms/content-camera.md:18]:
```suggestion
description: Use a content camera in Microsoft Teams Rooms that interacts with image-processing software to allow presenters to draw on an analog whiteboard.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested change, Many thanks.

File: Teams/rooms/content-camera.md (hunk 1 above)
Comment (hyoshioka0128) [reply Teams/rooms/content-camera.md:18]: @JohanFreelancer9 - Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**